keps/sig-node/4381-dra-structured-parameters/README.md

```diff
@@ -1620,14 +1620,15 @@ type QuotaSpec struct {
     // +optional
     AllowAdminAccess bool `json:"allowManagementAccess,omitempty"`
 
-    // Stretch goals for 1.31:
-    //
-    // - maximum number of devices matching a selector
-    // - maximum sum of a certain quantity attribute
-    //
-    // These are additional restrictions when checking whether a device
-    // instance can satisfy a request. Creating a claim is always allowed,
-    // but allocating it fails when the quota is currently exceeded.
+    // The total number of allocated devices matching these selectors must not be
+    // exceeded. This has to be checked in addition to other claim constraints
+    // when checking whether a device can be allocated for a claim.
+    MaxDeviceCounts []MaxDeviceCount
+
+    // The sum of some quantity attribute of allocated devices must not
+    // exceed a maximum value. This has to be checked in addition to other claim constraints
+    // when checking whether a device can be allocated for a claim.
+    MaxQuantity []MaxQuantity
```

Comment:
Should be `MaxQuantities`, but see below.

Reply:
I thought we were only going to do admin access for this release? Can we agree to that, we can add more quota features later if we have extra time (ha ha). I think getting into this debate now will delay us.

Reply:
I don't want to do class-based quota if that isn't going to be the final solution, so we have to agree on how to do quota.

If we "do it later", then we still need to know how we are going to add it and how it will interact with clients using the 3.11 API.

Reply:
Let's discuss in https://github.com/kubernetes-sigs/wg-device-management/issues/24, we have too many threads open in this PR regarding quota.

Reply:
I can take out this part again and move it into https://github.com/kubernetes-sigs/wg-device-management/issues/24.

Reply:
Ok, please do that. Resolving.